docs/getting-started/advanced-setting.asciidoc

```diff
@@ -67,6 +67,14 @@ IMPORTANT: {es-sec} requires {ecs-ref}[ECS-compliant data]. If you use third-par
 collectors to ship data to {es}, the data must be mapped to ECS.
 <<siem-field-reference>> lists ECS fields used in {es-sec}.
 
+[discrete]
+[[update-threat-intel-indices]]
+== Update default Elastic Security threat intelligence indices
+
+The `securitySolution:defaultThreatIndex` advanced setting allows users to specify one or more threat intelligence indices from which {elastic-sec} will collect threat match data. It controls features that query threat indices, such as the Threat Intelligence view on the Overview page, and uses the `filebeat-*`` index by default.
+
+Users can enable cross cluster search by adding threat indices from remote clusters. {filebeat} indices are supported. Non-{filebeat} indices require ECS-compliant threat field mappings to ensure threat matching works.
```

Comment:
> { filebeat } indices are supported.

This statement is potentially misleading; it could lead users to believe that any filebeat data will work as a threat index, which is _technically_ true but probably not what they want. I would either clarify that 

> { filebeat } indices containing threatintel module data are supported.

However, the sentence's location implies that this has to do with CCS, but it does not. Maybe we just need a few sentences describing the expected data format:

> Data in these indices is expected to contain { threat indicator } ECS fields, and to be otherwise ECS compatible. We recommend using the { filebeat threatintel } modules to generate this data, but you may also roll your own.